specs/signature-specification.md

```diff
@@ -127,21 +85,21 @@ A signature envelope consists of the following components:
 
 A signature envelope is `e = {m, v, u, s}` where `s` is signature.
 
-This specification defines the set of signed and unsigned attributes that make up a valid The Notary v2 signature. This specification aims to be be agnostic of signature envelope format (e.g. COSE, JWS), details of encoding the envelope in a specific signature envelope format are covered in in separate specs.
+This specification defines the set of signed and unsigned attributes that make up a valid Notary Project signature. This specification aims to be be agnostic of signature envelope format (e.g. COSE, JWS), details of encoding the envelope in a specific signature envelope format are covered in in separate specs.
 
-Notary v2 supports the following envelope formats:
+Notary Project signature supports the following envelope formats:
 
 - [JWS](./signature-envelope-jws.md)
 - [COSE Sign1](./signature-envelope-cose.md)
```

Comment:
COSE_Sign1 is one COSE type defined in COSE spec. There are other COSE types. I will remove it from signature specification, since it is described in `signature-envelope-cose`.